src/dune_patch/dune_patch.ml

```diff
@@ -15,18 +15,16 @@ include struct
 end
 
 let re =
-  let line xs = Re.seq ((Re.bol :: xs) @ [ Re.eol ]) in
-  let followed_by_line xs = Re.seq [ Re.str "\n"; line xs ] in
-  Re.compile
-  @@ Re.seq
-       [ Re.alt
-           [ line [ Re.str {|--- a/|}; Re.group (Re.rep1 Re.notnl) ]
-           ; line [ Re.str {|--- |}; Re.group (Re.rep1 Re.notnl) ]
-           ]
-       ; Re.alt
-           [ followed_by_line [ Re.str {|+++ b/|}; Re.group (Re.rep1 Re.notnl) ]
-           ; followed_by_line [ Re.str {|+++ |}; Re.group (Re.rep1 Re.notnl) ]
-           ]
+  let open Re in
```

Comment:
Can you remove the style changes? For example the local open, and the conversion of parens to `@@`. It makes it hard to understand the diff.